### PR TITLE
Fix/feat(chrome) Use latest chrome version 74

### LINKED ISSUE #456: chromedriver dependency broken in docker image v1.3
The chromedriver dependency in the latest docker image `v1.3` is broken, and errors out. Below is a stacktrace of the error.

```
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome version must be between 70 and 73
  (Driver info: chromedriver=73.0.3683.68 (47787ec04b6e38e22703e856e101e840b65afe72),platform=Linux 4.9.125-linuxkit x86_64)
```

I did not run into the same issue when I used an older version: `v1.1.0`

### PR DESCRIPTION
**Pull request by s-pace**
This PR:
- Update the chrome version being used. Use the latest 74
- Update the correct version of python being used (3.6)

resolves #456

### CODE REVIEW

**Comment by s-pace** [comment]
@pranav7 image already available with the user agent fix algolia/docsearch-scraper:v1.3.1

**Comment by pranav7** [comment]
@s-pace Nice one! 👍It works!!

File: scraper/dev/docker/Dockerfile.base [hunk 1]
```diff
@@ -52,8 +52,10 @@ COPY Pipfile .
...
 ENV LANG en_US.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
 ENV PIPENV_HIDE_EMOJIS 1
 RUN apt-get update -y && apt-get install -yq \
-    python-pip
-RUN pip install pipenv
-RUN pipenv install
\ No newline at end of file
+    python3-pip
```
Comment (BenoitPerrot) [inline scraper/dev/docker/Dockerfile.base]: this is unrelated to the commit and to the PR, please move to another

Comment (s-pace) [reply scraper/dev/docker/Dockerfile.base]: This is required by the PR since otherwise we do have the package import issue due to twisted

Comment (BenoitPerrot) [reply scraper/dev/docker/Dockerfile.base]: What about putting it in a separate commit?
